libdnf/CMakeLists.txt

```diff
@@ -78,6 +78,7 @@ target_link_libraries(libdnf
     ${OPENSSL_CRYPTO_LIBRARY}
     ${RPM_LIBRARIES}
     ${SCOLS_LIBRARIES}
+    ${SELINUX_LIBRARIES}
```

Comment:
I guess this should be linked just conditionally, like `IF (ENABLE_SELINUX)`, right?

Reply:
Good catch. Now the variable is undefined empty end expands to an empty string if SELinux was disabled. I will make the variable use conditional.